src/Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -24,11 +24,14 @@ file that was distributed with this source code.
 {%- endblock form_errors %}
 
 {% block sonata_help %}
-{% apply spaceless %}
-{% if sonata_help is defined and sonata_help %}
-    <span class="help-block sonata-ba-field-widget-help">{{ sonata_help|raw }}</span>
-{% endif %}
-{% endapply %}
+    {% apply spaceless %}
+        {% if sonata_help is defined and sonata_help %}
+            {% set translationDomain = sonata_admin.field_description.translationDomain ?: admin.translationDomain %}
+            <span class="help-block sonata-ba-field-widget-help">
+                {{ sonata_help|raw|trans({}, translationDomain)  }}
```

Comment:
The `|raw` implies that there could be HTML in there… are we really going to translate HTML? Also, isn't this a BC-break? Won't people get a lot of "missing translations" messages when getting your changes?

Reply:
> Won't people get a lot of "missing translations" messages when getting your changes?

This should be okay, because it's just a warning and has no visible change for the enduser.

Reply:
completely agree, my bad, sorry